src/specs/api.yaml

```diff
@@ -636,7 +636,20 @@ paths:
               schema:
                 type: object
                 properties: {}
+
       description: This path will create a new pipeline that can run a state machine with gem2s tasks.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                paramsHash:
+                  type: string
+              required:
+                - paramsHash
+        description: Request body for GEM2S request
+      parameters: []
```

Comment:
what is this `parameters: []` for?

Reply:
This is supposed to hold the list of parameters that will be used by the `POST` operation. We have defined the list of parameters that is used by the path on line 620. Having this property under `POST` will override it for `POST` operations, which is not good. I think this might have been automatically added by Stoplight Studio.

I have found this issue under other paths. I'll remove all of them. 

Reference in Swagger specification: https://swagger.io/specification/

![image](https://user-images.githubusercontent.com/2862362/135243194-289d5d98-cfe0-46e6-ac59-5ef13f379545.png)

Reply:
After discussing with Marcell, this is not a problem. The parameter configuration in the higher-level `PATH` object will not be replaced by the `parameters` that's defined in the lower level `POST` object. This seems to be a property that is set by Stoplight Studio. Right now the property is not doing anything so we can just leave it there. If someone modifies the schema again using Stoplight studio, it's going to be added again. Editing it manually might cause errors in the schema. So, it's best to just leave it. It's essentially harmless.

Reply:
ok